imagersite/imager_profile/admin.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib import admin
+
+from .models import ImagerProfile
+
+
+# class ImagerProfileAdmin(admin.ModelAdmin):
+#     fieldsets = [
+#         (None,               {'fields': ['user']}),
+#         ('Date information', {'fields': ['camera'], 'classes': ['collapse']}),
+#     ]
+#     list_display = ('user.username', 'camera', 'website_url')
```

Comment:
please don't let commented code stay in your source files.  Delete it and let git keep your history safe